code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -305,7 +305,7 @@
 
 /datum/gas_reaction/bzformation/init_reqs()
 	min_requirements = list(
-		/datum/gas/tritium = 10,
+		/datum/gas/nitrous_oxide = 10,
```

Comment:
This might be a little out of scope for this PR, but how do people feel about this change?
I don't want two reactions to have the same requirements, as it gets a bit confusing. Also, BZ is pretty hard to make as it is, and N2O is rarely used in reactions, so I don't see the issue with this change.